src/tests/ftest/nvme/pool_capacity.py

```diff
@@ -94,16 +94,28 @@ def run_test_create_delete(self, num_pool=2, num_cont=5, total_count=100):
             self.log.info("Running test %s", loop_count)
             offset = loop_count * num_pool
             for val in range(offset, offset + num_pool):
-                self.pool.append(self.get_pool(namespace="/run/pool_qty_{}/*".format(num_pool),
-                                 properties="reclaim:disabled"))
+                self.pool.append(
+                    self.get_pool(
+                        namespace="/run/pool_qty_{}/*".format(num_pool),
+                        properties="reclaim:disabled"))
 
                 display_string = "pool{} space at the Beginning".format(val)
                 self.pool[-1].display_pool_daos_space(display_string)
                 nvme_size_begin[val] = self.pool[-1].get_pool_free_space("NVME")
-                # To be fixed by DAOS-12974
+                # Before creating a container, we'll disconnect the pool. However, disconnecting the
+                # pool with threads causes the test to crash, so disconnect before creating any
+                # container. We connect during get_pool_free_space() above.
+                self.pool[-1].disconnect()
```

Comment:
I think it's worth re-verifying this after #14153 lands.

Reply:
I removed the disconnect and it works. Thanks.